README.md

```diff
@@ -12,12 +12,16 @@ Optionally, you may set the overlay to run when the physical switch on the displ
 To install dependencies, you will need [poetry](https://python-poetry.org/docs/#installation) installed on your computer.
 
 At this point in time, you'll also need Python 3.7 installed. Python 3.6 may also work (untested), but right now, the OpenCV version we are using does not have a build available for Python 3.8. [pyenv](https://github.com/pyenv/pyenv) may be used to quickly switch between python versions. To do this,
-- Install `pyenv`,
+- Install `pyenv` (easy installer [here](https://github.com/pyenv/pyenv-installer), and dependencies [here](https://github.com/pyenv/pyenv/wiki/Common-build-problems) (use Debian dependencies fo the Raspbery Pi)),
```

Comment:
While we're on the topic of clarifying the installation process in the README, should we break up the `# Installation` section up? Right now, I see:

1. Setup .env file
2. Modify config.json
3. Install poetry
4. Install pyenv
5. Setup OpenCV
6. Install libraries

The `.env` file part seems to be stashed away and not made clear in the README. Although, it isn't a crucial step because we do have default values in place.

Reply:
Actually we don't have default values for the `.env` in place, only for `config.json`. I was wondering if we should though.